src/components/SideNav/components/NavContent/NavContent.styles.ts

```diff
@@ -0,0 +1,13 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+export const useStyles = makeStyles(theme => ({
+  toolbar: {
+    ...theme.mixins.toolbar,
+    display: "flex",
+    alignItems: "center",
+    padding: "0 14px",
```

Comment:
instead of 14px maybe we should use the theme.spacing?

Reply:
Fxied